### ISSUE TITLE
France: still wrong number of confirmed cases (April 13)

### ISSUE THREAD

**Issue by alfkoehn** [CSSEGISandData/COVID-19#2173]
Number in database here: 136779

Number in other data sources [1,2]: 98076

37103 cases too much, probably due to also including _potential_ cases (and not only _confirmed_ cases), see e.g. the discussion [3] and an official statement [4].

[1] https://github.com/opencovid19-fr/data/blob/master/ministere-sante/2020-04-12.yaml
[2] https://dashboard.covid19.data.gouv.fr/
[3] https://github.com/CSSEGISandData/COVID-19/issues/2005
[4] https://github.com/CSSEGISandData/COVID-19/issues/2094

**Comment by JiPiBi** [comment]
You are right : in issue  2094 JHU announced that they will correct the time series , but they didn't ......

**Comment by yetzt** [comment]
most media have been reporting speculative cases as confirmed cased for over a week now.
